tools/lsp/preview/ui/property_view.rs

```diff
@@ -178,19 +180,166 @@ fn convert_number_literal(
     }
 }
 
+#[derive(Clone, Debug, Default)]
+struct ExtractedColor {
+    pub value: Option<slint::Color>,
+    pub name: Option<String>,
+}
+
+fn extract_color_from_evaluated_expression_impl(
+    evaluated_expression: &expression_tree::Expression,
+    mut palette_name: Vec<String>,
```

Comment:
not sure what that is, sometimes it is struct field, sometimes palette access. Could you add a comment?

Reply:
I stopped doing comments.

It's always the name of the color in the palette. Its something like `Global.Property.Struct1Field.Struct2Field`.

Reply:
Doc comments don't count:-) They are obviously needed just to keep clippy happy.